pkg/network/protocols/http2/protocol.go

```diff
@@ -343,6 +353,47 @@ func (p *protocol) GetStats() *protocols.ProtocolStats {
 	}
 }
 
+// UpdateKernelTelemetry should be moved to the HTTP/2 part as well
+func (p *protocol) UpdateKernelTelemetry(mgr *manager.Manager) {
+	var zero uint32
+
+	for {
+		mp, _, err := mgr.GetMap(probes.HTTP2TelemetryMap)
+		if err != nil {
+			log.Warnf("error retrieving http2 telemetry map: %s", err)
+			return
+		}
```

Comment:
that shouldn't be inside the for loop

Reply:
@amitslavin, still happening

Reply:
my bad, after adding the ticker logic , it changed the code logic. changed it again